cms/static/js/spec/views/modals/validation_error_modal_spec.js

```diff
@@ -64,23 +64,18 @@ define(['jquery', 'underscore', 'js/spec_helpers/validation_helpers', 'js/views/
                 ];
 
                 var callback = function() {
-                    return true;
+                    done();
                 };
 
                 // Show Modal and click undo changes
                 showModal(errorObjects, callback);
                 expect(ValidationHelpers.isShowingModal(modal)).toBeTruthy();
                 ValidationHelpers.undoChanges(modal);
+            });
 
-                // Wait for the callback to be fired
-                waitsFor(function () {
-                    return callback();
-                }, 'the callback to be called', 5000);
-
+            it('it hides modal after undo changes button is clicked', function () {
```

Comment:
Not sure why this is in a separate spec since `undoChanges()` is being called on line 73.